### ISSUE TITLE
Cannot find module 'get-funtion-location'

### ISSUE THREAD

**Issue by intellectus-rojiwon** [samchon/nestia#900]
## Bug Report
A dependency on the get-function-location library exists in `@nestia/core` > `WebSocketAdaptor`. However, that library is not specified in the package.json of `@nestia/core`, which is causing an error "Can not find module `get-function-location`" in the runtime environment.

I solved the problem by temporarily installing `get-function-location` into the project. The library is installed by `@nestia/sdk`, so if I had the `@nestia/sdk` library installed in the runtime environment, I would not have found this bug. However, `@nestia/sdk` is a documentation build tool and is suitable for devDependencies, which are deleted with the `npm prune --omit=dev` command.

In conclusion, you should explicitly require a dependency on `get-function-location` in the `@nestia/core` library.

@nestia/core ^3.1.3
@nestia/sdk ^3.1.3

**Comment by samchon** [comment]
Oops, I've forgotten to add dependency on `@nestia/core` too.

Will fix it ASAP.

**Comment by samchon** [comment]
Upgrade to v3.1.4, then be fixed. Thanks for reporting.
